src/app/pages/payment/international-payment-status/international-payment-status.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { InternationalPaymentStatusComponent } from './international-payment-status.component';
+import { ActivatedRoute } from '@angular/router';
+import {activatedRouteSpy} from 'src/app/helpers/spies';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('InternationalPaymentStatusComponent', () => {
+  let component: InternationalPaymentStatusComponent;
+  let fixture: ComponentFixture<InternationalPaymentStatusComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
+      declarations: [ InternationalPaymentStatusComponent ],
+      providers: []
+      //   { provide: ActivatedRoute, useValue: activatedRouteSpy }, ]
```

Comment:
@patrickf949, let's remove the commented code

Reply:
@CryceTruly done